backend/app/testbot/llm.py

```
@@ -0,0 +1,32 @@
+    }
+    headers = {"Content-Type": "application/json"}
+    response = requests.post(GEMINI_API_URL, headers=headers, json=payload)
+    if response.status_code == 200:
+        try:
+            result = response.json()
+            if "candidates" in result and len(result["candidates"]) > 0:
+                candidate = result["candidates"][0]
+                if "content" in candidate and "parts" in candidate["content"]:
+                    answer_text = candidate["content"]["parts"][0].get("text", "").strip()
+                    return answer_text if answer_text else "No output returned."
+                else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)